libcxx/test/std/containers/sequences/array/size_and_alignment.compile.pass.cpp

```diff
@@ -0,0 +1,142 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// <array>
+
+// template <class T, size_t N>
+// struct array
+
+// Make sure std::array<T, N> has the correct object size and alignment.
```

Comment:
Only partially related but...

Any chance we've changed the triviality of any of the special members with this change?

Reply:
That's a great observation! This is tested in https://github.com/llvm/llvm-project/blob/main/libcxx/test/libcxx/containers/sequences/array/triviality.pass.cpp and I think this already has the appropriate coverage, but I am happy to add to it if you think we're missing something there.